config/default.js

```diff
@@ -267,28 +267,64 @@ module.exports = {
       bootChallenger:
         process.env.BOOT_CHALLENGER_ADDRESS || '0xfCb059A4dB5B961d3e48706fAC91a55Bad0035C9',
     },
-    tokens: [
-      {
-        name: 'MockERC20',
-        address: '0xB5Acbe9a0F1F8B98F3fC04471F7fE5d2c222cB44',
-        amount: 200,
-      },
-      {
-        name: 'Test-Eth',
-        address: '0x3f152B63Ec5CA5831061B2DccFb29a874C317502',
-        amount: '10000000000000000000000',
-      },
-      {
-        name: 'MATIC',
-        address: '0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae',
-        amount: '10000000000000000000000',
-      },
-      {
-        name: 'USDC',
-        address: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
-        amount: '1000000000000',
-      },
-    ],
+    tokens: {
+      blockchain1: [
+        {
+          name: 'MockERC20',
+          address: '0xB5Acbe9a0F1F8B98F3fC04471F7fE5d2c222cB44',
+          amount: 200,
+        },
+        {
+          name: 'Test-Eth',
+          address: '0x3f152B63Ec5CA5831061B2DccFb29a874C317502',
+          amount: '10000000000000000000000',
+        },
+        {
+          name: 'MATIC',
+          address: '0x499d11E0b6eAC7c0593d8Fb292DCBbF815Fb29Ae',
+          amount: '10000000000000000000000',
+        },
+        {
+          name: 'USDC',
+          address: '0x07865c6e87b9f70255377e024ace6630c1eaa37f',
+          amount: '1000000000000',
+        },
+      ],
+      mainnet: [
```

Comment:
Why aren't USDT and DAI in the list of restricted tokens?

Reply:
The browser doesn't support them as far as I know.

Reply:
I can add them though if you like, it just increases the deployment cost a little bit.